tox.ini

```
@@ -25,7 +30,8 @@ deps =
 deps =
     # sphinx
     Sphinx==4.2.0
-    django-dbdiff
+    ; django-dbdiff
+    git+https://github.com/pfouque/django-dbdiff.git@fix312#egg=django-dbdiff
```

To be rollback after merging/releasing https://github.com/yourlabs/django-dbdiff/pull/21

```suggestion
    django-dbdiff
```